time|level|timeoffset|message|location
071515_155346.878|INFO|000.000462| Software version: Plus-2.3.0.4217 - Linux| in vtkPlusLogger(183)
071515_155346.878|INFO|000.000541| Logging at level 3 (INFO) to file: /home/snr/Support/Plus-build-codeblocks/bin/Output/071515_155346_PlusLog.txt| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/PlusServer/Testing/PlusServer.cxx(85)
071515_155346.882|INFO|000.004570| EmTracker: Local time offset: 0ms| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/DataCollection/vtkPlusDevice.cxx(869)
071515_155346.882|ERROR|000.004720| Failed to open port: /dev/ttyS0 - Error opening serial connection| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/DataCollection/PolarisTracking/vtkNDITracker.cxx(228)
071515_155346.882|ERROR|000.004753| EmTracker: Cannot connect to data source, ConnectInternal failed| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/DataCollection/vtkPlusDevice.cxx(960)
071515_155346.882|ERROR|000.004770| Unable to connect device: EmTracker.| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/DataCollection/vtkDataCollector.cxx(306)
071515_155346.882|ERROR|000.004794| Datacollector failed to connect to devices| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/PlusServer/Testing/PlusServer.cxx(132)
